infrastructure/main.tf

```diff
@@ -6,11 +6,20 @@ locals {
   preview_vault_name     = "${var.raw_product}-aat"
   default_vault_name     = "${var.raw_product}-${var.env}"
   vault_name             = "${(var.env == "preview" || var.env == "spreview") ? local.preview_vault_name : local.default_vault_name}"
+
+  preview_vault_rg       = "${var.raw_product}-aat"
+  default_vault_rg       = "${var.raw_product}-${var.env}"
+  vault_rg               = "${(var.env == "preview" || var.env == "spreview") ? local.preview_vault_rg : local.default_vault_rg}"
```

Comment:
previously pull request would create(or look for) vault in its own resource group. This fixes it.

Reply:
isn't that exactly the same as `vault_name` block definition few lines above?

Reply:
After the change in line 17 it happens to be the same.
But the name of the vault and the resource group that it's in are two separate things and can be named differently.